REFERENCE.md

```diff
@@ -4456,15 +4456,15 @@ Password to use for StackStorm authentication.
 
 ### <a name="st2upgrade_mongodb"></a>`st2::upgrade_mongodb`
 
-The default upgrade for this plan goes from 3.4 to 3.6 and ultimately to 4.0
+The default upgrade for this plan goes from 3.4 to 3.8 and ultimately to 4.0
 
 High level steps:
 - stop stackstorm
-## https://docs.mongodb.com/manual/release-notes/3.6-upgrade-standalone/
+## https://docs.mongodb.com/manual/release-notes/3.8-upgrade-standalone/
```

Comment:
This section should not be changed, it's for upgrade of MongoDB 3.6 and not Python 3.6